pkg/apis/migration/v1alpha1/migtoken_types.go

```diff
@@ -24,7 +26,8 @@ type MigTokenSpec struct {
 // MigTokenStatus defines the observed state of MigToken
 type MigTokenStatus struct {
 	Conditions
-	ObservedDigest string `json:"observedDigest,omitempty"`
+	ExpiresIn      time.Time `json:"expiresIn,omitempty"`
```

Comment:
should we name this `ExpiresAt`? I tried to keep it consistent with oauthaccesstoken API but it is in seconds and this is in UTC so doesn't really make sense

Reply:
updated the name so it reads better :)